plugin/src/main/java/io/snyk/languageserver/protocolextension/SnykExtendedLanguageClient.java

```diff
@@ -122,12 +117,27 @@ public boolean getSastEnabled() {
     @JsonNotification(value = "$/snyk.hasAuthenticated")
     public void hasAuthenticated(HasAuthenticatedParam param) {
         var prefs = Preferences.getInstance();
-        prefs.store(Preferences.AUTH_TOKEN_KEY, param.getToken());
-        triggerScan(null);
-
-        if (!param.getToken().isBlank()) {
-            showAuthenticatedMessage();
-            enableSnykViewRunActions();
+        
+        var oldToken = prefs.getAuthToken();
+        var oldApi = prefs.getEndpoint();
+        
+        if (param.getApiUrl() != null && !param.getApiUrl().isBlank() && !param.getApiUrl().equals(oldApi)) {
+        	prefs.store(Preferences.ENDPOINT_KEY, param.getApiUrl());
```

Comment:
Balloon that endpoint was updated

Reply:
Should be sent from LS

Reply:
in that case we need to delete the notification from VS. I'm fine with both.